docs/GitlabCI.md

```diff
@@ -0,0 +1,47 @@
+# Running dockertest in Gitlab CI
```

Comment:
I think this would work well in the README! Would you be open to copy the contents there? :)

Reply:
Done)